package.json

```diff
@@ -98,6 +98,9 @@
       "<rootDir>/example/node_modules",
       "<rootDir>/lib/"
     ],
+    "moduleNameMapper": {
+      "@eppo(.*)": "<rootDir>/node_modules/@eppo/$1"
```

Comment:
Helps jest import from the eppo common library

Reply:
Interseting!